spec/support/api_request_helpers.rb

```diff
@@ -0,0 +1,44 @@
+module Spec
+  module Support
+    module ApiRequestHelpers
+      def get(path, **args)
```

Comment:
don't these come from rspec-rails automatically?  Curious why these are being added?

Reply:
process includes the request_headers (as defined by api_basic_authorize). Otherwise, each get/post will require the :headers => .... for the BasicAuth.  That is an option if you prefer and would shrink a bit some of the helper code.

Reply:
Let's discuss in a follow-up...I'm more concerned that "overwriting" these methods will cause confusion.